bin/docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM       fedora:22
+
+# install the required dependencies to compile native extensions
+RUN        dnf -y install gcc-c++ make ruby-devel libxml2-devel libxslt-devel findutils git ruby
+RUN        dnf -y install wget # needed by the website to fetch resources
```

Comment:
I'll do it but the wget for example is not required by the hibernate version of this.